tests/python/relax/test_op_manipulate.py

```diff
@@ -58,6 +58,7 @@ def test_reshape_infer_struct_info():
     s0 = relax.Var("s", R.Shape((3, 8, 5)))
     s1 = relax.Var("s", R.Shape(ndim=3))
     s2 = relax.Var("s", R.Shape())
+    s3 = relax.ShapeExpr((3, 8, 5))
```

Comment:
The tests can directly go to the unity branch I think, since the tests in this PR is pure enhancement 🤔 Would appreciate if you can send them to the unity branch, so that we won’t lose these tests after switch over.